src/aosm/HISTORY.rst

```diff
@@ -11,4 +11,8 @@ Release History
 
 1.0.0b2
 ++++++++
-* Fixed: Use default_factory when a dataclass default is hashable (Python 3.11 compatibility)
\ No newline at end of file
+* Fixed: Use default_factory when a dataclass default is hashable (Python 3.11 compatibility)
+
+1.0.0b3
```

Comment:
It is suggested to list version in descending order in HISTORY.rst. In other words, put the latest version in the top of the file, e.g., https://github.com/Azure/azure-cli-extensions/blob/106a571fb4529a47d97bbe566ca6aa0dcd55b090/src/containerapp/HISTORY.rst

Reply:
Done, thanks.